### DISCUSSION TITLE
Restore DB from backup

### DISCUSSION THREAD

**Discussion by unintended-consequences** [Chia-Network/chia-blockchain#15107] [Support]
I have used `chia db backup --no_indexes --backup_file /path/to/backup.sqlite`

to create regular backups of my blockchain database. How do I actually restore this backup to a functional chia installation? Is it just copy the file to `~/.chia/mainnet/db/blockchain_v2_mainnet.sqlite`?

**Comment by unintended-consequences** [comment]
Well, copying the file didn't work.

**Reply by iTZUNAMI** [reply]
yes just copy the big db file. You can download it also from website / downloads under all linux version

**Reply by erytrmn** [reply]
Hello when i copy big db file my chia's full node section only the circles are spinning.What i need to do for fix this?

**Reply by iTZUNAMI** [reply]
copy, restart the farmer, wait, can take some hours to completly sync. i've done a lot of times, windows or linux it's the same

**Reply by BrandtH22** [reply]
Hey @unintended-consequences and @iTZUNAMI , as a heads up we currently can provide more immediate and thorough support in our discord server (https://discord.gg/chia).

As for restoring after using the chia db backup feature (unintended-consequences) you should only need to:
1. Close the chia client
2. Rename the backed up db file to the proper name (should be just removing the "Vacuumed_" prefix) and ensure it is in the proper directory (default location is correct but essentially make sure it is in the db folder)
3. Restart the chia client

Note - I just tested and verified the above steps on my machine, if you do run into issues do not hesitate to ask in our discord server (https://discord.gg/chia)

As for using the chia torrent (iTZUNAMI) you will need to:
1. Download the torrent from the official site: https://www.chia.net/downloads/
2. Assemble the torrent file using a torrent software (I use the classic BitTorrent)
3. Rename the db to "blockchain_v2_mainnet.sqlite" (should just need to remove the torrent and date reference) and ensure it is in the proper directory (default location is correct but essentially make sure it is in the db folder)
4. Restart the chia client

If you do run into issues do not hesitate to ask in our discord server (https://discord.gg/chia)
